docs/processes/quality-assurance/git-contribution-guide.md

```diff
@@ -173,10 +173,11 @@ Pull requests (draft and otherwise) are created from the GitHub website. Further
 
 Pull requests require **a minimum of two approvals**. The default approach is to
 choose a reviewer from your team for the first review. However, the reviewer may
-be from different team, for example a domain expert in Ruby or C++. Depending on
-the team size and dynamics, the required approvals can be higher or lower.
-Please try to have at least another person to look at the changes beside the
-author.
+be from different team, for example a domain expert in a programming language,
+quality assurance process, telemetry or documentation. Depending on the team
+size and dynamics, the number of required approvals can be higher or lower.
+Think about the tradeoffs between velocity with quality when deciding on an
+approval process for your team.
```

Comment:
I do like how this refers to a domain expert - I think this provides the clarity we are after surrounding reviewers. This also addresses team size issues.

I think once the people start getting engaged with PRs and GitHub as a whole we will see a happy medium with quality and velocity. I also think this will tip toward velocity closer to the deadline.
As long as we raise awareness about our expectations this should improve throughout the companies life span. 
- highlighting good examples
- showing areas of improvement
- ect.

We want to empower our employees, thus we should also trust they will educate themselves accordingly

Reply:
Thank you for the feedback @talzid. 

> I do like how this refers to a domain expert 

I have some ideas to bring the skill matrix -> domain expert -> review automation into action :)

> We want to empower our employees, thus we should also trust they will educate themselves accordingly

Well put 👏🏼